tests/app/notifications/test_process_notifications.py

```diff
@@ -1057,3 +1064,157 @@ def test_send_notification_without_sms_sender_rate_limit_uses_regular_delivery_t
 
     assert mocked_chain.call_args[0][0].task == 'deliver_sms'
     deliver_sms_with_rate_limiting.assert_not_called()
+
+
+@mock_aws
+def test_send_notification_to_queue_delayed(client, mock_sqs, mocker, sample_notification) -> None:
+    """Test send_notification_to_queue_delayed happy path"""
+    # create queue for testing
+    sqs_client, q_url = mock_sqs
+
+    notification: Notification = sample_notification()
+
+    debug_logger = mocker.spy(client.application.logger, 'debug')
+
+    assert (
+        send_notification_to_queue_delayed(
+            notification=notification,
+            research_mode=False,
+            queue_name='test_queue',
+            sms_sender_id=None,
+            delay_seconds=0,
+        )
+        is None
+    )
+
+    debug_logger.assert_called_once()
+    start = monotonic()
+
+    # MaxNumberOfMessages ranges from 1-10, want to ensure only one message was added to the queue
+    messages = sqs_client.receive_message(QueueUrl=q_url, MaxNumberOfMessages=10)
+    assert len(messages.get('Messages')) == 1
+    assert (monotonic() - start) < 1
+
+
+@mock_aws
+def test_send_notification_to_queue_delayed_delays_properly(client, mock_sqs, mocker, sample_notification) -> None:
+    """Test send_notification_to_queue_delayed happy path with delay value"""
+    # create queue for testing
+    sqs_client, q_url = mock_sqs
+
+    notification: Notification = sample_notification()
+
+    debug_logger = mocker.spy(client.application.logger, 'debug')
+
+    assert (
+        send_notification_to_queue_delayed(
+            notification=notification,
+            research_mode=False,
+            queue_name='test_queue',
+            sms_sender_id=None,
+            delay_seconds=1,
+        )
+        is None
+    )
+
+    debug_logger.assert_called_once()
+
+    # MaxNumberOfMessages ranges from 1-10, want to ensure only one message was added to the queue
+    messages = {}
+    start = monotonic()
+    while messages.get('Messages') is None:
+        messages = sqs_client.receive_message(QueueUrl=q_url, MaxNumberOfMessages=10)
+        # prevent infinite loop if nothing is added to the queue
+        if monotonic() - start > 2:
+            break
+
+    assert len(messages.get('Messages')) == 1
+
+    # verify it took at least 1 sec to get message
+    delay = monotonic() - start
+    assert delay > 1
```

Comment:
Please mention delay_seconds in the comment and/or modify the code to use delay_seconds. 

nit:
delay can be calculated and asserted on the same line